test/coverlet.core.tests/Instrumentation/InstrumenterTests.cs

```diff
@@ -187,6 +223,8 @@ class InstrumenterTest
             public string Identifier { get; set; }
 
             public DirectoryInfo Directory { get; set; }
+
+            public Mock<ILogger> Logger { get; set; }
```

Comment:
I don't see any reference to this prop...it's a typo or I'm missing something?

Reply:
Done. Sorry missed to delete the property before pushing.